python/tk_multi_loader/delegate_publish_history.py

```diff
@@ -84,9 +204,7 @@ def _on_before_paint(self, widget, model_index, style_options):
         # v004 (2014-02-21 12:34)
 
         header_str = ""
-
-        if sg_item.get("version_number"):
-            header_str += "<b style='color:#2C93E2'>Version %03d</b>" % sg_item.get("version_number")
+        header_str += "<b style='color:#2C93E2'>Version %03d</b>" % (sg_item.get("version_number") or 0)
```

Comment:
Could we put this on the technical debt list and see if we can do something for 1.0? This is so error prone than even a Toolkit veteran got it wrong. Not a good sign.